mvpa/cosmo_classify_lda.m

```diff
@@ -138,7 +142,7 @@
                                 'in training']);
                 end
                 nfirst=n; % keep track of number of samples
-            elseif nfirst~=n
+            elseif opt.check_data & nfirst~=n
```

Comment:
You're right, I added this in as I only looked at rank-ordered decision values. It is not necessary.